src/Domain/ContentRenderer/CodeRenderer.php

```diff
@@ -47,4 +48,22 @@ private function getLanguageClasses(): array {
 		return [ 'language-' . htmlspecialchars( $this->language ) ];
 	}
 
+	/**
+	 * @return (string|string[])[]
+	 */
+	private function getWrapperAttributes( string $contentUrl ): array {
+		$attributes = [];
+
+		$attributes['class'] = $this->getWrapperClasses();
+
+		$attributes['data-toolbar-order'] = 'copy-to-clipboard';
+
+		if ( $this->showEditButton == true ) {
```

Comment:
No need to do a comparison for booleans. But also when doing comparisons, prefer using the explicit `===`.